test/integration/basic_nsmdp_test.go

```diff
@@ -23,15 +24,15 @@ func TestNSMDDP(t *testing.T) {
 
 	Expect(err).To(BeNil())
 
-	k8s.Prepare("nsmd", "icmp")
+	k8s.Prepare("dataplane", "icmp", "nsmgr")
```

Comment:
Apparently, we have a couple of these left all over. I'll fix these in a separate PR.